util/util.go

```diff
@@ -0,0 +1,291 @@
+	}
+
+	return blobsDir
+}
+
+func IsBlobReflectionOff() bool {
+	return os.Getenv("REFLECT_BLOBS") == "false"
+}
+
+func GetLBRYNetDir() string {
+	lbrynetDir := os.Getenv("LBRYNET_DIR")
+	if lbrynetDir == "" {
+		usr, err := user.Current()
+		if err != nil {
+			log.Errorln(err.Error())
+			return ""
+		}
+		return usr.HomeDir + "/.lbrynet/"
+	}
+	return lbrynetDir
+}
+
+const ALL = true
+const ONLINE = false
+
+func GetLBRYNetContainer(all bool) (*types.Container, error) {
+	return getDockerContainer("lbrynet", all)
+}
+
+func getDockerContainer(name string, all bool) (*types.Container, error) {
+	cli, err := client.NewEnvClient()
+	if err != nil {
+		panic(err)
+	}
+	filters := filters.NewArgs()
+	filters.Add("name", name)
+	containers, err := cli.ContainerList(context.Background(), types.ContainerListOptions{All: all, Filters: filters})
+	if err != nil {
+		panic(err)
+	}
+	for _, container := range containers {
+		fmt.Printf("%s %s\n", container.ID[:10], container.Image)
+	}
+	if len(containers) == 0 {
+		return nil, nil
+	}
+	if len(containers) > 1 {
+		return nil, errors.Err("more than one %s container found", name)
+	}
+
+	return &containers[0], nil
+}
+
+func IsUsingDocker() bool {
+	return os.Getenv("LBRYNET_USE_DOCKER") == "true"
+}
+
+func IsRegTest() bool {
+	return os.Getenv("REGTEST") == "true"
+}
+
+func GetLbrycrdClient(lbrycrdString string) (*lbrycrd.Client, error) {
+	var lbrycrdd *lbrycrd.Client
+	var err error
+	if lbrycrdString == "" {
+		lbrycrdd, err = lbrycrd.NewWithDefaultURL()
+		if err != nil {
+			return nil, err
+		}
+	} else {
+		lbrycrdd, err = lbrycrd.New(lbrycrdString)
+		if err != nil {
+			return nil, err
+		}
+	}
+
+	return lbrycrdd, nil
+}
+
+func IsCleanOnStartup() bool {
+	return os.Getenv("CLEAN_ON_STARTUP") == "true"
+}
+
+func IsLbrynetRunning() (bool, error) {
+	if IsUsingDocker() {
+		container, err := GetLBRYNetContainer(ONLINE)
+		if err != nil {
+			return false, err
+		}
+		return container != nil, nil
+	}
+
+	processes, err := ps.Processes()
+	if err != nil {
+		return true, errors.Err(err)
+	}
+	var daemonProcessId = -1
+	for _, p := range processes {
+		if p.Executable() == "lbrynet" {
+			daemonProcessId = p.Pid()
+			break
+		}
+	}
+
+	running := daemonProcessId != -1
+	return running, nil
+}
+
+func CleanForStartup() error {
+	if !IsRegTest() {
+		return errors.Err("never cleanup wallet outside of regtest and with caution. this should only be done in local testing and requires regtest to be on")
+	}
+
+	running, err := IsLbrynetRunning()
+	if err != nil {
+		return err
+	}
+	if running {
+		err := StopDaemon()
+		if err != nil {
+			return err
+		}
+	}
+
+	err = CleanupLbrynet()
+	if err != nil {
+		return err
+	}
+
+	lbrycrd, err := GetLbrycrdClient(os.Getenv("LBRYCRD_STRING"))
+	if err != nil {
+		return errors.Prefix("error getting lbrycrd client: ", err)
+	}
+	height, err := lbrycrd.GetBlockCount()
+	if err != nil {
+		return errors.Err(err)
+	}
+	if height < 110 {
```

Comment:
magic number

Reply:
We need 10 LBC to do work most of the time, and you cannot have spendable inputs until 100 blocks. I will add a const for this so its more descriptive.